packages/api/package.json

```diff
@@ -11,10 +11,10 @@
     "build": "tsc",
     "debug:bin": "NODE_OPTIONS=--no-warnings node --loader ts-node/esm src/bin.ts",
     "lint:types": "tsc --noEmit",
-    "prebuild": "rm -rf dist/; npm run version",
+    "prebuild": "rm -rf dist/ && npm run version",
```

Comment:
semicolons are pretty problematic — if the preceding command fails then it moves onto the next one 😬